xmtp_mls/src/groups/mod.rs

```diff
@@ -65,14 +68,26 @@ pub enum GroupError {
     WelcomeError(#[from] openmls::prelude::WelcomeError<StorageError>),
     #[error("client: {0}")]
     Client(#[from] ClientError),
-    #[error("receive errors: {0:?}")]
-    ReceiveError(Vec<MessageProcessingError>),
+    #[error("receive error: {0}")]
+    ReceiveError(#[from] MessageProcessingError),
+    #[error("Receive errors: {0:?}")]
+    ReceiveErrors(Vec<MessageProcessingError>),
     #[error("generic: {0}")]
     Generic(String),
     #[error("diesel error {0}")]
     Diesel(#[from] diesel::result::Error),
 }
 
+impl crate::retry::RetryableError for GroupError {
+    fn is_retryable(&self) -> bool {
+        match self {
```

Comment:
This is a reasonable starting place. We'll have to do some diving into the OpenMLS error types to figure out which ones can be added to the list as a follow-on. For example, some of their internal error types wrap keystore errors, which are actually database errors and should be retried.

Reply:
Yeah, I wasn't sure which errors to mark here, i'll do some diggin and update